app/src/main/java/ELME/Model/Graph.java

```diff
@@ -31,4 +31,21 @@ public void evaluateImpl() {
         nodes.forEach(node -> node.evaluate());
     }
 
+    public ArrayList<Node> getNodes() {
+        return nodes;
+    }
+
+    public void setNodes(ArrayList<Node> nodes) {
+        this.nodes = nodes;
+    }
+
+    public boolean isIsCollapsed() {
```

Comment:
IsCollapsed should be enough for the name of the method

Reply:
Oh, yepp. Will correct that. Pesky auto-generation....